tests/FileLockTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+$fname = 'tests/lock-test.txt';
```

Comment:
Should be a part of test class or variable in test method.

Reply:
understood